iris_grib/message.py

```diff
@@ -437,6 +455,7 @@ def _get_key_value(self, key):
                        'satelliteNumber', 'instrumentType',
                        'scaleFactorOfCentralWaveNumber',
                        'scaledValueOfCentralWaveNumber',
+                       'longitude', 'latitude',
                        'longitudes', 'latitudes')
```

Comment:
Note that, removing `'longitudes', 'latitudes'` results in the tests still working, but I'm nervous about doing this as the whole foundation of this PR is unclear.

Reply:
Totally agree.
I think this is the safest approach